lnrpc/routerrpc/router.proto

```diff
@@ -589,7 +590,7 @@ message PaymentStatus {
     repeated lnrpc.HTLCAttempt htlcs = 4;
 }
 
-message CircuitKey {    
+message CircuitKey {
```

Comment:
weird...looks like a previous commit is missing `make rpc-format`?

Reply:
Double check your proto versions? Running `make rpc` on the current master branch doesn't make any changes.

Reply:
you need run `make rpc-format`, not `make rpc`.